src/Exceptionless.Api/Startup.cs

```diff
@@ -39,8 +39,12 @@ public class Startup {
             app.UseAuthentication();
             app.UseMiddleware<ProjectConfigMiddleware>();
             app.UseMiddleware<RecordSessionHeartbeatMiddleware>();
-            // Throttle api calls to X every 15 minutes by IP address.
-            app.UseMiddleware<ThrottlingMiddleware>();
+
+            if (Settings.Current.ApiThrottleLimit >= 0 && Settings.Current.ApiThrottleLimit != int.MaxValue) {
+                // Throttle api calls to X every 15 minutes by IP address.
```

Comment:
We shouldn't be checking for -1 just Int32.MaxValue; We need to normalize -1 in the settings.